internal/v4/auth_test.go

```diff
@@ -123,7 +123,7 @@ func (s *commonSuite) testMacaroonAuth(c *gc.C, p httptesting.JSONCallParams) {
 		"is-authenticated-user ",
 	})
 	checkedCaveats = nil
-	c.Assert(cookieJar.cookieURLs, gc.DeepEquals, []string{"http://somehost/"})
+	c.Assert(cookieJar.cookieURLs, gc.DeepEquals, []string{"http://somehost/v4/log"})
```

Comment:
I think we need to rejig the test somewhat. The URL is not the interesting part of the cookie.